Projects/UOContent/Gumps/RunebookGump.cs

```diff
@@ -7,504 +7,448 @@
-                                            $"{yLat}° {yMins}'{(ySouth ? "S" : "N")}, {xLong}' {xMins}'{(xEast ? "E" : "W")}";
-                                        from.SendMessage(location);
-                                    }
-
-                                    Book.OnTravel();
-                                    new RecallSpell(from, e, Book, Book).Cast();
-
-                                    Book.Openers.Remove(from);
-                                }
+                                _book.OnTravel();
+                                new RecallSpell(from, e, _book, _book).Cast();
 
-                                break;
+                                _book.Openers.Remove(from);
                             }
-                        case 1: // Drop rune
+
+                            break;
+                        }
+                    case 1: // Drop rune
+                        {
+                            if (!_book.IsLockedDown || from.AccessLevel >= AccessLevel.GameMaster)
                             {
-                                if (!Book.IsLockedDown || from.AccessLevel >= AccessLevel.GameMaster)
-                                {
-                                    Book.DropRune(from, e, index);
+                                _book.DropRune(from, e, index);
 
-                                    from.CloseGump<RunebookGump>();
-                                    if (!Core.ML)
-                                    {
-                                        from.SendGump(new RunebookGump(from, Book));
-                                    }
-                                }
-                                else
+                                from.CloseGump<RunebookGump>();
+                                if (!Core.ML)
                                 {
-                                    Book.Openers.Remove(from);
-
-                                    from.SendLocalizedMessage(
-                                        502413,
-                                        null,
-                                        0x35
-                                    ); // That cannot be done while the book is locked down.
+                                    from.SendGump(new RunebookGump(_book));
                                 }
+                            }
+                            else
+                            {
+                                _book.Openers.Remove(from);
 
-                                break;
+                                from.SendLocalizedMessage(
+                                    502413,
+                                    null,
+                                    0x35
+                                ); // That cannot be done while the book is locked down.
                             }
-                        case 2: // Set default
+
+                            break;
+                        }
+                    case 2: // Set default
```

Comment:
This case doesn't remove openers. It's actually a bug! I think we should remove openers no matter what.

Reply:
it's not actually a bug, actually - there are two cases where it re-sends the gump and therefore they are still openers